views/layout.html

```diff
@@ -79,22 +79,22 @@ <h1 class="microsite-name">
         </header>
 
         <div class="filter-bar filter-bar--hidden clearfix js-filter-bar" id="filterbar">
-            <form class="search-form js-search-form">
+            <form class="search-form js-search-form" >
                 <div class="search-form__inner">
-                    <label for="tipue_drop_input" class="visuallyhidden">
+                    <label for="tipue_drop_input" class="visuallyhidden" >
                         Search for a book title or author
                     </label>
-                    <input type="text" class="o-forms-text search-form__field js-tipue-drop" autocomplete="off" id="tipue_drop_input" placeholder="Search for a book title or author" value="" />
-                    <label type="submit" value="Submit" class="search-form__submit" tabindex="-1" for="tipue_drop_input"></label>
-                    <div id="tipue_drop_content"></div>
+                    <input type="text" class="o-forms-text search-form__field js-tipue-drop" autocomplete="off" id="tipue_drop_input" placeholder="Search for a book title or author" value="" data-trackable="search-box" />
+                    <label type="submit" value="Submit" class="search-form__submit" tabindex="-1" for="tipue_drop_input" ></label>
+                    <div id="tipue_drop_content" ></div>
                 </div>
             </form>
             <form class="filter-form js-filter-form" action="{{ router.url('home') }}" method="get">
                 <div class="filter-form__inner clearfix">
-                    <label for="filter" class="filter-form__label text-quiet text-uppercase text-small">
+                    <label for="filter" class="filter-form__label text-quiet text-uppercase text-small" >
                         Category
                     </label>
-                    <select name="category" id="filter" class="filter-form__select o-forms-select js-filter-form-select">
+                    <select name="category" id="filter" class="filter-form__select o-forms-select js-filter-form-select" data-trackable="category-dropdown">
```

Comment:
I think we can remove the data-trackable here now and just rely on the custom event that we're sending?